Scheduler/Task.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Scheduler
+{
+	/// <summary>
+	/// Task Table Info
+	/// </summary>
+	public class Task
+	{
+		public long TaskId { get; set; }
+
+		[ForeignKey("Goal")]
+		
+		public long? TaskParentId { get; set; }
+
+		public string Name { get; set; }
+
+		public int Status { get; set; }
```

Comment:
Make this field of enum type (first create corresponding enum)

Reply:
**Review2**: what about this?
I suggest to make two statuses for now: created and done